docs/tests/unit-tests.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Unit Tests
+---
+
+# Unit Tests
+
+A unit test is about testing in isolate a single function and all of its possible output.
+
+Unit tests can be found in
+
+`spec/models` and other various directories within the `spec`
+directory.
+
+`spec/system` is for integration tests
+
+`spec/requets` is for acceptance tests
```

Comment:
You explained how integration and acceptance tests are not unit tests, so they shouldn't probably be mentioned here.

BTW: most tests in spec/ are functional tests, those testing units in isolation are really few